src/table.scss

```diff
@@ -418,6 +418,24 @@ $block: #{$fd-namespace}-table;
     justify-content: space-between;
   }
 
+  &__column-header-icons {
+    display: flex;
+
+    fd-icon:not(:last-child) {
+      margin-right: 0.25rem;
```

Comment:
Could point to the specs from where you took this value?

Reply:
Inna, this spacing is not documented anywhere, I've applied the one that I see fit: icons look too stacked, if no margin is used.

Interestingly, in [SAPUI5 SDK](https://sapui5.hana.ondemand.com/#/entity/sap.ui.comp.smarttable.SmartTable/sample/sap.ui.comp.sample.smarttable) icons are separated with space 
![image](https://user-images.githubusercontent.com/33101123/134858360-06fac264-8ec5-42f0-9553-0806b2b9c396.png)

Should I just keep it as-is or what will be the best here?

Reply:
I am also not seeing this documented in the specs so I think it is fine to make a general assumption until we hear otherwise